src/HaskellCI/Project.hs

```diff
@@ -7,31 +7,35 @@ module HaskellCI.Project (
     parseProjectFile,
     ) where
 
-import           Data.ByteString                              (ByteString)
-import           Data.Generics.Labels                         ()
-import           GHC.Generics                                 (Generic)
-import           Lens.Micro                                   (over)
+import  Data.Bifoldable      (Bifoldable (..))
+import  Data.Bifunctor       (Bifunctor (..))
+import  Data.Bitraversable   (Bitraversable (..), bifoldMapDefault, bimapDefault)
+import  Data.ByteString      (ByteString)
+import  Data.Generics.Labels ()
+import  GHC.Generics         (Generic)
+import  Lens.Micro           (over)
 
 import qualified Data.Map.Strict                              as M
 import qualified Distribution.CabalSpecVersion                as C
 import qualified Distribution.FieldGrammar                    as C
+import qualified Distribution.Fields.Pretty                   as C
 import qualified Distribution.PackageDescription.FieldGrammar as C
 import qualified Distribution.Parsec.Common                   as C
 import qualified Distribution.Parsec.Newtypes                 as C
 import qualified Distribution.Parsec.Parser                   as C
 import qualified Distribution.Parsec.ParseResult              as C
 import qualified Distribution.Types.SourceRepo                as C
-import qualified Distribution.Fields.Pretty as C
 
-import           HaskellCI.Newtypes
-import           HaskellCI.Optimization
-import           HaskellCI.ParsecError
+import HaskellCI.Newtypes
+import HaskellCI.Optimization
+import HaskellCI.ParsecError
 
 -- $setup
 -- >>> :seti -XOverloadedStrings
 
-data Project a = Project
+data Project b a = Project
```

Comment:
Is there a reason that these type parameters are in the order `b a` rather than `a b`? It's not terribly important in the grand scheme of things, but it does look a bit strange to me.

Reply:
no really, but as you can see from smart coloring of GitHub diff, this way has its merits ;)